app/components/back-office-app-services/bo-appeals-casedata-import-function/function-app.tf

```diff
@@ -32,3 +32,9 @@ resource "azurerm_servicebus_subscription" "register_fo_casedata_subscription" {
   topic_id           = var.service_bus_appeals_fo_appellant_submission_id
   max_delivery_count = 1
 }
+
+resource "azurerm_servicebus_subscription" "register_fo_lpaq_subscription" {
+  name               = "register-fo-lpaq-subscription"

```

Comment:
I think we should be consistent with naming. Use submission instead of register?

Register was used for BO applications for subscriptions I think, in this case we call it lpa submission elsewhere.

`fo-lpaq-submission-subscription` perhaps? would need to match in code of course.